main.cc

```diff
@@ -0,0 +1,82 @@
+#include <fmt/core.h>
+#include <cmath>
+#include <complex>
+#include <cstdlib>
+#include <iostream>
+#include <vector>
+
+typedef std::complex<float> cmplxf;
```

Comment:
This is a sensible typedef for C++ – no need for having `lv_32fc_t` in C++ land, right?